Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'rails', '5.2.2'
-gem 'json', '~> 1.8'
+gem 'json', '~> 2.1'
```

Comment:
__[RuboCop]__ Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `json` should appear before `rails`.

<sub>posted by Sider</sub>